docs/operators/cache.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "@cache"
+sidebar_position: 7
+---
+
+The **@cache** operator enables caching for the query, field or type it is applied to. For eg:
+
+```graphql
+schema {
+  query: Query
+}
+
+type Query {
+  posts: [Post] @cache(maxAge: 3000)
+  user(id: Int): User
+}
+
+type Post {
+  title: String
+  body: String
+  user: User
+}
+
+type User @cache(maxAge: 4000) {
+  id: Int
+  name: String @cache(maxAge: 8000)
```

Comment:
lets add `## maxAge` heading to this doc similar to other operators with explanation on the parameter and the unit is used to specify it.

Reply:
added a small description for `maxAge`